.circleci/config.yml

```diff
@@ -62,7 +62,14 @@ references:
   filters_release_build: &filters_release_build
     tags:
       only:
-        - /^v\d+\.\d+\.\d+(?:-\w+\.\d+)?$/
+        - /^v\d+\.\d+\.\d+$/
+    branches:
+      ignore: /.*/
+  
+  filters_prerelease_build: &filters_release_build
```

Comment:
**issue:** the CI is failing because the anchor name hasn't been updated after pasting

```suggestion
  filters_prerelease_build: &filters_prerelease_build
```

Reply:
omg my god i was staring at the config for so long trying to work out what was wrong